scaletest/templates/scaletest-runner/main.tf

```diff
@@ -625,8 +625,6 @@ resource "coder_agent" "main" {
     vscode     = false
     ssh_helper = false
   }
-  startup_script_timeout  = 86400
-  shutdown_script_timeout = 7200
   startup_script_behavior = "blocking"
```

Comment:
```suggestion
```

Reply:
'kind of deprecated' is not the same as actually deprecated; I'll leave this one as-is and we can deprecate in a separate PR.